lib/basedriver/driver.js

```diff
@@ -62,6 +61,10 @@ class BaseDriver extends MobileJsonWireProtocol {
     this._eventHistory = {
       commands: [] // commands get a special place
     };
+
+    this.log = logger.getLogger(
+      () => `${this.constructor.name}` +
+            (_.isString(this.sessionId) && this.sessionId.length >= 8) ? ` [${this.sessionId.substring(0, 8)}]` : '');
```

Comment:
Parentheses are mismatched in here. This is breaking the unit tests, too. (Why are the unit tests not run locally? This would be found.)

Reply:
Fixed